actionview/lib/action_view/helpers/form_tag_helper.rb

```diff
@@ -117,17 +117,43 @@ def field_id(object_name, method_name, *suffixes, index: nil, namespace: nil)
         ].tap(&:compact!).join("_")
       end
 
+      # Generate the text-only content that would be used for a the given name and
+      # field combination.
+      #
+      #   field_label :post, :title
+      #   # => "Title"
+      #
+      #   field_label :post, :published, value: true
+      #   # => "Published"
+      #
+      #   field_label :post, :published, value: false
+      #   # => "Draft"
+      #
+      def field_label(object_name, method_name, object: nil, value: nil)
+        object_name, method_name = object_name.to_s.dup, method_name.to_s.dup
+        object ||= instance_values[object_name]
```

Comment:
This is cherry-picked from `ActionView::Helpers::Tags::Base`. I'd love to re-use that code instead of copy it, but I wasn't sure how to best achieve that without sizable changes.